azext_edge/edge/_help.py

```diff
@@ -1353,7 +1395,51 @@ def load_iotops_help():
         - name: Add an issuer certificate to the OPC UA Broker's issuer certificate list with custom secret name.
           text: >
             az iot ops connector opcua issuer add --instance instance --resource-group instanceresourcegroup
-            --certificate-file "certificate.der" --secret custom-secret-name
+            --certificate-file "certificate.der" --secret-name custom-secret-name
+        - name: Add an issuer certificate to issuer certificate list and skip overwrite confirmation prompt when secret already exists.
```

Comment:
Add an issuer certificate to **the** issuer certificate list

Reply:
addressed in last commit

Reply:
<img width="350" alt="image" src="https://github.com/user-attachments/assets/15fe2200-1805-49b8-a739-ce14de929e73">